src/trivia/util.h

```diff
@@ -855,8 +855,22 @@ illegal_instruction(void)
 #ifdef ENABLE_ASAN
 # include <sanitizer/lsan_interface.h>
 # define LSAN_IGNORE_OBJECT(ptr) __lsan_ignore_object(ptr)
+
+/**
+ * Disable leak sanitizer. Leak check will not be performed on Tarantool
+ * exit.
+ */
+void
+lsan_turn_off();
+
 #else
 # define LSAN_IGNORE_OBJECT(ptr) ((void)ptr)
+
+static inline void
+lsan_turn_off()
```

Comment:
Nit: Should be declared as `lsan_turn_off(void)`.

Reply:
Fixed.

```diff
diff --git a/src/trivia/util.h b/src/trivia/util.h
index f861ec045c..a75c92b5ea 100644
--- a/src/trivia/util.h
+++ b/src/trivia/util.h
@@ -861,13 +861,13 @@ illegal_instruction(void)
  * exit.
  */
 void
-lsan_turn_off();
+lsan_turn_off(void);

 #else
 # define LSAN_IGNORE_OBJECT(ptr) ((void)ptr)

 static inline void
-lsan_turn_off()
+lsan_turn_off(void)
 {
 }
```